core/client/hdfs/pom.xml

```diff
@@ -38,6 +38,7 @@
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-client</artifactId>
+      <scope>provided</scope>
```

Comment:
the quick decision is to shade the hadoop dep on the client side

Reply:
I think it cannot work if we relocation the hadoop-client, we should use the hadoop from env.